frontend/src/modules/filters/organizer/connector.ts

```diff
@@ -0,0 +1,14 @@
+import { FilterWithoutType } from '../interface';
+import { adaptOrganizerFilter } from './adapter';
+import { fetchOrganizer } from './api';
+
+export const getOrganizerFilter = async (language: string): Promise<FilterWithoutType | null> => {
+  let results = null;
+  try {
+    const rawOrganizer = await fetchOrganizer({ language });
+    results = rawOrganizer.results;
+  } catch (e) {
+    // error
+  }
+  return adaptOrganizerFilter(results);
```

Comment:
Something like this maybe? Haven't tested it but looks more readable

```suggestion
return adaptOrganizerFilter(
	await fetchOrganizer({ language })
	.catch(e => null)
);
```

Reply:
indeed, it's more readable.
The main difference: in the current version, we break the asynchronism by waiting for the result before returning it.
I think your version is also ok but this needs to be tested.